cli/utils.py

```diff
@@ -25,6 +25,10 @@ def dict_insert(dic, val, key, *keys):
     :param key: first key in a chain of key that will store the value
     :param keys: sub keys in the keys chain
     """
+    # case of processing sub-key and parent-key has not been given
```

Comment:
This check is required for optional yaml arguments. I would like to keep that change to be more safe.